client/src/components/Projects/ProjectNew/TableMember/TableMember.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import { Table } from "antd";
+import "./style.css";
+import filterData from "./logic";
+
+export default class TableMember extends Component {
+  constructor(props) {
+    super(props);
+    this.state.member = require("./member.json");
+    this.state.filterArray = filterData(this.state.member);
+    this.columns[0].filters = this.state.filterArray[1];
+    this.columns[1].filters = this.state.filterArray[2];
```

Comment:
**The constructor for a React component is called before it is mounted.**

so if your request to get the member take time your page will not load till it finished
which mean you can't implement loading.. indicator to tell the user that your page is fetching data from the server

**Also, ** `this.columns[0].filters` will not be modified if the state changed for some reason which means you have to refresh the page ("unmount then mount the component") to apply the changes to the filters

I'd advise fetching the data and updating the state (`this.setState({})) on `componentDidMount`
Also, assigning the filters inside the `render` function, so the filters will be updated when the `state` changes